### ISSUE TITLE
Main menu seem to be broken

### ISSUE THREAD

**Issue by User-green** [woheller69/weather#29]
**Describe the bug**
Broken symbols in main menu.

**How to Reproduce**
1. Use "russian" as phone local
2. Install and set up app as usual
3. Oprn main menu

**Expected behavior**
Symbols is not broken

**Screenshots**
![image30.png](https://user-images.githubusercontent.com/52010870/130406976-c0015b96-375d-4874-9053-15335d1f4c73.png)
**Environment:**
 - App version: 4.1 (F-droid lastest)
 - Android version: 8.1.0, Flyme 7.1.4RU
 - Device: Meizu 16 (M872H)

**Comment by woheller69** [comment]
When switching to Russian I cannot see this. 
Latest F-Droid version is 4.3.

**Comment by User-green** [comment]
> When switching to Russian I cannot see this. 
I have no clue about this. 
> Latest F-Droid version is 4.3.
Yep, you right. I got updated, but bug last.

**Comment by woheller69** [comment]
Can you manually scroll to the top?

Have you tried uninstalling?

**Comment by woheller69** [comment]
Can you try Privacy Friendly Weather? Would be good to know if you see this bahaviour there as well

**Comment by User-green** [comment]
> Can you manually scroll to the top?
> Have you tried uninstalling?

Yes to bouth. Havent helped.

> Can you try Privacy Friendly Weather? Would be good to know if you see this bahaviour there as well

It doesnt have this issue. However, it also doesnt feature RU menu lang.

**Comment by woheller69** [comment]
Ok. Will compare with Privacy Friendly Weather code next week. 
When does it happen?

When reloading data?
When switching between tabs?
When opening the app?

**Comment by shishakohle** [comment]
Sorry to ask a silly one: Which symbols exactly do u find to be broken in your screenshot?

**Comment by woheller69** [comment]
The weather symbol should be fully visible - scrolled to the top when opening the app, reloading or switching between tabs. On my Android 6, 10, and 11 devices which I use for testing it does...

**Comment by woheller69** [comment]
Another question. What happens if you allow display rotation and hold the phone in landscape mode? Do you get 2 columns, scrolled to the top?

**Comment by shishakohle** [comment]
@User-green is your issue the weather symbol not being visible when scrolled to the top / opening the app? Or is your issue the squares (rain probability) mistaken as "broken symbols"?

**Comment by woheller69** [comment]
Good point. Maybe we are talking about different things. The squares are correct. When rain is coming they will be filled as defined in the legend below.
